### PR TITLE
Dependency bumps

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: basket/news/migrations/0027_alter_newsletter_requires_double_optin.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.2.19 on 2023-05-10 17:06
```
Comment (pmac) [inline basket/news/migrations/0027_alter_newsletter_requires_double_optin.py:1]: Why is this necessary?

**Review by pmac — APPROVED** [review]
Looks good. Not sure why there is a migration, but assuming it's needed this is good to go ✅

File: basket/news/migrations/0027_alter_newsletter_requires_double_optin.py (hunk 1 above)
Comment (robhudson) [reply basket/news/migrations/0027_alter_newsletter_requires_double_optin.py:1]: String change in `help_text` to add a space after the period. Kinda lame but 🤷‍♂️ .

Comment (stevejalim) [reply basket/news/migrations/0027_alter_newsletter_requires_double_optin.py:1]: Agreed @robhudson  - I'm not a fan Django needing a new migration when tweaks to, essentially, Python strings have to happen. In the past, when am sure it's a no-op change in terms of real DB schema, I've dug out the most recent migration to touch the field and edited that old migration to match, so that Django no longer sees the change. It's a little risky because it might mess with a git-bisect if someone needs to dig into history, but it can work if the planets are appropriately aligned
